src/scss/atomic/molecules/_grid.scss

```diff
@@ -104,12 +104,12 @@
     @extend %disable-flex-grow;
     justify-content: space-between;
 
-    @include min-desktop {
-        [class*="flexgrid__cell"]:nth-child(3n+1):nth-last-child(-n+2) ~ [class*="flexgrid__cell"] {
-            margin-right: auto;
-            margin-left: $gridGutterSize;
-        }
-    }
+    //@include min-desktop {
```

Comment:
I removed the code all together, because the space it defines should be reserved by the grid gutter? Its weird to do something with a nth-child if you are not aware how many items there are on a row...

Reply:
I fixed grid--gutter so space-between can be used if there is space needed between the elements ;-)